demos/chatroom/experiment.py

```diff
@@ -1,11 +1,19 @@
 """Coordination chatroom game."""
 
 import dallinger as dlgr
+from dallinger.config import get_config
 try:
     unicode = unicode
 except NameError:  # Python 3
     unicode = str
 
+config = get_config()
+
+
+def extra_settings():
+    config.register('network', unicode)
+    config.register('n', int)
```

Comment:
We should add documentation for the `extra_settings` hook, and also be sure to mention the change in release notes when we make a release. (We can hopefully justify the pain of changing existing experiments by pointing out that it makes it possible to check setting types at startup rather than at runtime.)